controller/controller.go

```diff
@@ -230,6 +231,7 @@ func (c *Controller) RunOnce(ctx context.Context) error {
 	verifiedARecords.Set(float64(vARecords))
 	verifiedAAAARecords.Set(float64(vAAAARecords))
 	endpoints, err = c.Registry.AdjustEndpoints(endpoints)
+	endpoints = endpoint.RemoveDuplicates(endpoints)
```

Comment:
Should this happen before L226? Otherwise it seems the count might be incorrect, though I'm not sure what the consequence of that is.

```go
	sourceEndpointsTotal.Set(float64(len(endpoints)))
```

Reply:
Nice catch! I didn't see that, I need to check if AdjustEndpoints generates the duplicates or the duplicates are at L220. I have an easy way to see that, let me verify